java/src/main/native/src/map_lookup.cu

```diff
@@ -127,24 +128,55 @@ get_gather_map_for_map_values(column_view const &input, string_scalar &lookup_ke
 } // namespace
 
 namespace jni {
-std::unique_ptr<column> map_lookup(column_view const &map_column, string_scalar lookup_key,
-                                   bool has_nulls, rmm::cuda_stream_view stream,
-                                   rmm::mr::device_memory_resource *mr) {
-  // Defensive checks.
+
+
+void map_check(column_view const &map_column, rmm::cuda_stream_view stream) {
```

Comment:
Originally I want to make this method `get_structs_column` that returns the structs_column in line 137.
But I got 

```
error: no suitable user-defined conversion from "cudf::column_view" to "std::unique_ptr<cudf::column, std::default_delete<cudf::column>>" exists
```
And I don't know how to fix it.
So I just rename it to map_check, and return nothing.

Reply:
Done. put it to default namespace{}